### ISSUE TITLE
Import scss from parent directories

### ISSUE THREAD

**Issue by lonix1** [refgd/easy-complie#24]
None of these work:

````scss
@import "node_modules/bootstrap/scss/_functions.scss";
@import "/node_modules/bootstrap/scss/_functions.scss";
@import "~bootstrap/scss/_functions.scss";
@import "../../../../../node_modules/bootstrap/scss/_functions.scss";
````

Error is `File not found`.

What is the correct way to import from a parent directory?

**Comment by refgd** [comment]
Fix on 1.1.6

**Comment by lonix1** [comment]
@refgd Thanks!
